.eslintrc.json

```diff
@@ -0,0 +1,31 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "airbnb"
+  ],
+  "globals": {
+    "Atomics": "readonly",
+    "SharedArrayBuffer": "readonly"
+  },
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "plugins": [
+    "react"
+  ],
+  "rules": {
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }],
+    "react/prefer-stateless-function": [1, { "ignorePureComponents": true }],
+    "import/no-unresolved": "off",
+    "no-shadow": "off"
+  }
+}
```

Comment:
**[ REQUIRED ]** File is still different as the requirement please do copy the content of the file [HERE](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) and paste it on your file overwriting the content.